tests/test_fixtures/custom_pandas_dataset.py

```diff
@@ -113,3 +113,9 @@ def expect_column_to_start_with_valid_drg(self, column):
     @PandasDataset.column_map_expectation
     def expect_column_values_to_have_odd_lengths(self, column):
         return column.map(lambda x: len(x) % 2 == 1)
+
```

Comment:
Hmm... again -- Dataset is a `v012` type.

Reply:
All the fixes in this PR are for pre-013 API.